File: abc/093/b.rb
a, b, k = gets.strip.split.map(&:to_i)

list = []

(1..k).each do |i|
  list << a + i - 1
  list << b - i + 1
end

sorted = list.select { |i| a <= i && i <= b }.uniq.sort

sorted.each do |i|
  puts i
end

File: abc/093/b.py
a, b, k = map(int, input().strip().split())
result = set()

for small in range(a, min(a + k, b + 1)):
    result.add(small)

for large in range(max(a, b - k + 1), b + 1):
    result.add(large)

result = sorted(list(result))

for r in result:
    print(r)
